index.html

```diff
@@ -0,0 +1,101 @@
+    />
+    <title>Personal portfolio</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <ul class="list">
+        <li><a href="#">About</a></li>
+        <li><a href="">Work</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="section-content">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section class="section-project">
+      <h2 class="header-section">These are some of my projects</h2>
+
+      <div class="section-grid">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx"
+          ><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"
+            alt=""
+            class="section-images"
+          />
+          <p class="">Tribute Page</p></a
+        >
+        <a href="https://codepen.io/freeCodeCamp/full/qRZeGZ"
+          ><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png"
+            alt=""
+            class="section-images"
+          />
+          <p>Random Quote Machine</p>
+        </a>
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX"
+          ><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png"
+            alt=""
+            class="section-images"
+          />
+          <p>JavaScript Calculator</p></a
+        >
+        <a href="https://codepen.io/freeCodeCamp/full/mVEJag"
+          ><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg"
+            alt=""
+            class="section-images"
+          />
+          <p>Map Data Across Globe</p></a
+        >
+        <a href="https://codepen.io/freeCodeCamp/full/wGqEga"
+          ><img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png"
+          alt="" class="section-images" /
+          <p>Wikipedia Viewer</p></a
+        >
+        <a href="https://codepen.io/freeCodeCamp/full/KzXQgy"
+          ><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png"
+            alt=""
+            class="section-images"
+          />
+          <p>Tic Tac Toe Game</p></a
+        >
+      </div>
+      <div class="btn">
+        <button class="btn-1">
+          <a href="https://codepen.io/FreeCodeCamp/">Show all</a>
+        </button>
+      </div>
+    </section>
+    <footer class="footed">
+      <div class="sub-footer">
+        <h2>Let's work together...</h2>
+        <p>How do you take your coffee?</p>
+        <div class="sub-footer-link">
+          <a href="https://www.facebook.com/freecodecamp">Facebook</a>
+          <a href="https://github.com/freecodecamp">GitHub</a>
+          <a href="https://twitter.com/freecodecamp">Twitter</a>
+          <a href="mailto: freecodeCamp.co">Send a Mail</a>
+          <a href="tel:+4733378901">Call Me</a>
+        </div>
+      </div>
+    </footer>
```

Comment:
This is not a footer section, this is a contact section, kindly fix that and add the footer section to your project.

Reply:
- [ ] Kindly always make sure you explain why the changes are suggested :smile: 
`HINT: This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*`